src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-list/cephfs-subvolume-list.component.html

```diff
@@ -1,22 +1,42 @@
-<ng-container *ngIf="subVolumes$ | async as subVolumes">
-  <cd-table [data]="subVolumes"
-            columnMode="flex"
-            [columns]="columns"
-            selectionType="single"
-            [hasDetails]="false"
-            (fetchData)="fetchData()"
-            (updateSelection)="updateSelection($event)">
+<div class="row">
+  <div class="col-sm-2">
+    <h2 i18n>Groups</h2>
+    <ng-container *ngIf="subVolumeGroups$ | async as subVolumeGroups">
+      <ul class="nav flex-column nav-pills">
+        <li class="nav-item">
+          <a class="nav-link"
+             [class.active]="'' === subVolumeGroupName"
+             (click)="selectSubVolumeGroup('')">Default</a>
+        </li>
+        <li class="nav-item"
+            *ngFor="let subVolumeGroup of subVolumeGroups">
+          <a class="nav-link"
```

Comment:
We could add a [word-break](https://getbootstrap.com/docs/5.0/utilities/text/#word-break) class here and it'll break the word into next line whenever the name is too long.

```suggestion
          <a class="nav-link text-break"
```